### PR TITLE
Exception Handling in C++

### LINKED ISSUE #6957: Exception Handling in C++
### Description

Video about handling Exceptions in C++.

### Domain

C/CPP

### Type of Contribution

Video

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by saniya44**
<hr>

## Description 📜

C_CPP:- Video of Exception Handling in C++ #6957 issue was assigned to me. I did my best and added VIDEO_Exception Handling in C++ folder into Exception Handling directory of C_CPP branch.
Fixes #6957 

<hr>

## Type of change 📝

- [x] Video (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)

<hr>

## Domain of Contribution 📊

- [x] C/CPP

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Review by Suvraneel — CHANGES_REQUESTED** [review]
@saniya44  Kindly stick to `.md` format to maintain uniformity across this domain...
Remove the `.ipynb` file and add those examples as codeblocks in the md itself

Also, readme.md is reserved here for indexes only... rename your 1st file

After that, mentors will review...

**Comment by saniya44** [comment]
> @saniya44  Kindly stick to `.md` format to maintain uniformity across this domain...
> Remove the `.ipynb` file and add those examples as codeblocks in the md itself
> 
> Also, readme.md is reserved here for indexes only... rename your 1st file
> 
> After that, mentors will review... 

Okay 👍

**Comment by saniya44** [comment]
Done requested changes.

File: C_CPP/Exception Handling/VIDEO_Exception Handling in C++/SourceCode.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+```
+1. Exception.cpp
+//Simple Exception Handling- Divide by 0
+#include <iostream>
+using namespace std;
```
Comment (Suvraneel) [inline C_CPP/Exception Handling/VIDEO_Exception Handling in C++/SourceCode.md:5]:
```suggestion
1. Exception.cpp
```cpp
//Simple Exception Handling- Divide by 0
#include <iostream>
using namespace std;

```

File: C_CPP/Exception Handling/VIDEO_Exception Handling in C++/SourceCode.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    }
+    catch(int i){
+        cout<<"\n Divide by zero error!! Number can not be divide by 0. Error number "<<i;
+    }
+    return 0;
+}
+
+2. Exception2.cpp
+//Exception Handling using multiple throw and catch
+#include <iostream>
```
Comment (Suvraneel) [inline C_CPP/Exception Handling/VIDEO_Exception Handling in C++/SourceCode.md:28]:
```suggestion
}
```

## 2. Exception2.cpp
```cpp
//Exception Handling using multiple throw and catch
#include <iostream>

```

File: C_CPP/Exception Handling/VIDEO_Exception Handling in C++/SourceCode.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    }
+    catch (float i)
+    {
+        cout << "Caught Float Exception value=" << a;
+    }
+    return 0;
+}
+
+3. Exception3.cpp
+//Multiple throw single catch block
```
Comment (Suvraneel) [inline C_CPP/Exception Handling/VIDEO_Exception Handling in C++/SourceCode.md:65]: Same

**Review by Suvraneel — CHANGES_REQUESTED** [review]
@saniya44  Observe changes made to video link file...

In the source code, put every code in a separate codeblock
& use codeblock highlighting , like this  ` ```cpp `

**Comment by Suvraneel** [comment]
image
@saniya44  Also, these many commits aren't desirable... Use Git Bash... or GitHub desktop even... Dont use the Web Interface

**Comment by saniya44** [comment]
Okay 👍

**Comment by saniya44** [comment]
Okay 👍

**Comment by saniya44** [comment]
Closed this cause of so many mistakes...
Refer correct one here: https://github.com/girlscript/winter-of-contributing/pull/8427

**Pull request closed without merging**